wlc/__init__.py

```diff
@@ -704,8 +704,10 @@ def upload(self, file, overwrite=None, **kwargs):
         """Updoad a translation file to server."""
         self.ensure_loaded("file_url")
         url = self._attribs["file_url"]
-        files = {"file": file}
-
+        if kwargs.get("format"):
+            files = {"file": (f"file.{kwargs.get('format')}", file)}
```

Comment:
Let's make the code simpler:

```suggestion
        if "format" in kwargs:
            files = {"file": (f"file.{kwargs.['format']}", file)}
```